docs/data/charts/gauge/gauge.md

```diff
@@ -4,7 +4,7 @@ title: React Gauge chart
 
 # Charts - Gauge 🚧
 
-<p class="description">Gauge chart allows to evaluate a metric.</p>
+<p class="description">Gauge charts allow to evaluate a metric.</p>
```

Comment:
@alexfauquette I used the plural on all the chart pages to be consistent

Reply:
@LukasTy I let the plural here to be coherent and remove the "allows to" but we can do a follow up to clean all those descriptions